src/components/modals/login-modal/LoginContent.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Form, Input } from 'antd';
+import React from 'react';
+import './login-content.less';
+
+interface Props {
+  switchToForgotPasswordPage: () => void;
+}
+
+const LoginContent: (props: Props) => JSX.Element = (props: Props) => {
+  const forgotPasswordLink = (
+    <p onClick={props.switchToForgotPasswordPage} className="page-link">
+      here
+    </p>
+  );
+
+  const signUpLink = <a className="page-link">here</a>; // eslint-disable-line no-eval
```

Comment:
Using <a> tags without a href isn't very good practice, we should style this with a different tag (either text or button)